pkg/serverconfig/types.go

```diff
@@ -6,14 +6,16 @@ package serverconfig
 
 // Config is the top-level console server cli configuration.
 type Config struct {
-	APIVersion    string `yaml:"apiVersion"`
-	Kind          string `yaml:"kind"`
-	ServingInfo   `yaml:"servingInfo"`
-	ClusterInfo   `yaml:"clusterInfo"`
-	Auth          `yaml:"auth"`
-	Customization `yaml:"customization"`
-	Providers     `yaml:"providers"`
-	Helm          `yaml:"helm"`
+	APIVersion     string `yaml:"apiVersion"`
+	Kind           string `yaml:"kind"`
+	ServingInfo    `yaml:"servingInfo"`
+	ClusterInfo    `yaml:"clusterInfo"`
+	Auth           `yaml:"auth"`
+	Customization  `yaml:"customization"`
+	Providers      `yaml:"providers"`
+	Helm           `yaml:"helm"`
+	MonitoringInfo `yaml:"monitoringInfo,omitempty"`
+	LoggingInfo    `yaml:"loggingInfo,omitempty"`
```

Comment:
I'd just leave out `omitempty`. It's not on anything else, and we never serialize this. We only read it in.